src/js/view/preview.js

```diff
@@ -29,6 +38,26 @@ Object.assign(Preview.prototype, {
         style(this.el, {
             backgroundImage: backgroundImage
         });
+       
+        if (this.hasZoomThumbnail) {
+            this.enableZoomThumbnail();     
+        }
+    },
+    enableZoomThumbnail: function() {
+        if (this.model.get('isFloating')) {
+            return;
+        }
+        
```

Comment:
You could also run `clearTimeout(this.zoomThumbnailTimeout);`  so there can only ever be one active `zoomThumbnailTimeout`.